### PR TITLE
added guidance regarding updating single instances start and end date…

### PR DESCRIPTION
**Pull request by baywet**
…s in recurrence patterns
Some context: https://stackoverflow.com/questions/32771787/adding-exceptions-to-recurring-meeting-in-office365-apiusing-rest

### CODE REVIEW

**Comment by baywet** [comment]
anyone? CC @Lauragra @angelgolfer-ms

**Comment by Lauragra** [comment]
I made some edits to the proposed text. We should have someone review/verify technical accuracy. Thanks!

**Comment by Lauragra** [comment]
@angelgolfer-ms , okay to merge this PR?

**Comment by angelgolfer-ms** [comment]
@baywet , I apologize for the delay. We've been heads-down for the upcoming Build Conference. I just asked the feature owner to review your note. @Lauragra or myself will get back as soon as we hear back. Again, sorry for the delay.

**Comment by angelgolfer-ms** [comment]
@baywet , I'm relaying the input from the feature owner:

1)	In the documentation on the UpdateEvent verb, the paragraph should be moved to the Response section of the page
2)	The edit should be reverted from the documentation on the PatternedRecurrence type
3)	The paragraph should be modified to not imply that this error is limited to the Daily pattern

A more accurate phrasing would be something like:

An HTTP 400 Bad Request response with an error code of “ErrorOccurrenceCrossingBoundary” indicates that the update violates an Outlook restriction on recurrence exceptions: an occurrence may not be moved to or before the day of the previous occurrence, nor may it be moved to or after the day of the following occurrence.

Do you mind making the changes? 
Thanks.

**Comment by baywet** [comment]
@angelgolfer-ms  thanks for relaying. All done. Hopefully the PR is ready to be merged now :)

File: api-reference/beta/api/event_update.md [hunk 1]
```diff
@@ -68,7 +65,9 @@ add, update, or delete your own app-specific data in custom properties of an ext
 
 ## Response
 
-If successful, this method returns a `200 OK` response code and updated [event](../resources/event.md) object in the response body.
+If successful, this method returns a `200 OK` response code and updated [event](../resources/event.md) object in the response body.  
+
+**Note:** An HTTP 400 Bad Request response with an error code of `ErrorOccurrenceCrossingBoundary` and the following error message: `Modified occurrence is crossing or overlapping adjacent occurrence` indicates that the update violates an Outlook restriction on recurrence exceptions: an occurrence may not be moved to or before the day of the previous occurrence, nor may it be moved to or after the day of the following occurrence.
```
Comment (angelgolfer-ms) [inline api-reference/beta/api/event_update.md]: Hi @baywet ,
Thanks for making the update. Can I suggest breaking up the sentence a bit for readability? The note is currently one long sentence.
>>
This method can return an `HTTP 400 Bad Request` response with an error code of `ErrorOccurrenceCrossingBoundary` and the following error message: 
"Modified occurrence is crossing or overlapping adjacent occurrence". 
This indicates the update has violated an Outlook restriction on recurrence exceptions - an occurrence may not be moved to or before the day of the previous occurrence, nor may it be moved to or after the day of the following occurrence.

Thank you!

Comment (baywet) [reply api-reference/beta/api/event_update.md]: Done!

**Comment by baywet** [comment]
Thanks @lauragra  and @angelgolfer-ms for your collaboration !

**Comment by angelgolfer-ms** [comment]
@baywet, thank you for your patience and continued support!

**Pull request merged**